File: ejemplos/calcular-edad/codigo.js
exports.main = async (event, callback) => {
  const offsetMinutes = -5 * 60;
  const fecha_de_nacimiento = new Date(parseInt(event.inputFields['fecha_de_nacimiento']) + offsetMinutes * 60);
  const hoy_0 = new Date();
  const hoy = new Date(hoy_0.getTime() + offsetMinutes * 60 * 1000);
  let años = hoy.getFullYear() - fecha_de_nacimiento.getFullYear();
  let meses = hoy.getMonth() - fecha_de_nacimiento.getMonth();
  if (meses < 0) {
    años--;
    meses += 12;
  }
  const parts = [];
  if (años > 0) {
    parts.push(`${años} año${años !== 1 ? 's' : ''}`);
  }
  if (meses > 0) {
    parts.push(`${meses} mes${meses !== 1 ? 'es' : ''}`);
  }
  callback({
    outputFields: {
      edad: parts.length > 0 ? parts.join(' y ') : '0 meses'
    }
  });
}

File: ejemplos/calcular-edad/codigo.py
from datetime import datetime, timezone, timedelta

def main(event):
  utc_minus_5 = timezone(timedelta(hours=-5))
  fecha_de_nacimiento = datetime.fromtimestamp(int(event["inputFields"]["fecha_de_nacimiento"])/1000, tz=utc_minus_5)
  hoy = datetime.now(utc_minus_5)
  años = hoy.year - fecha_de_nacimiento.year
  meses = hoy.month - fecha_de_nacimiento.month
  if meses < 0:
    años = años - 1
    meses = meses + 12
  partes = []
  if años > 0:
    partes.append(f"{años} año{'s' if años != 1 else ''}")
  if meses > 0:
    partes.append(f"{meses} mes{'es' if meses != 1 else ''}")
  return {
    "outputFields": {
      "edad": " y ".join(partes) if partes else "0 meses"
    }
  }